src/test/java/LoginPageTest.java

```diff
@@ -0,0 +1,51 @@
+import config_loaders.PropertiesLoader;
+import config_loaders.EnvironmentConfigLoader;
+import config_loaders.UserConfigLoader;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import pages.HomePage;
+import pages.LoginPage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class LoginPageTest {
+    private LoginPage loginPage;
+    private HomePage homePage;
+    private static WebDriver driver;
+    // I know that this is too long - will work with this tomorrow; also know that I can load properties in better way, also work with this tomorrow
+    private static EnvironmentConfigLoader environmentConfigLoader = new EnvironmentConfigLoader(new PropertiesLoader().load("environment"));
```

Comment:
 Why does new PropertiesLoader().load("environment") part is called here, not in EnvironmentConfigLoader constructor?